package.json

```diff
@@ -26,7 +26,9 @@
     "gulp-mocha-phantomjs": "^0.8.1",
     "mocha": "^2.2.5",
     "chai": "^3.2.0",
-    "gulp-jscs": "~2.0.0"
+    "gulp-jscs": "~2.0.0",
+    "requirejs-plugins": "^1.0.2",
```

Comment:
@buunguyen This will need to be a reference to my github branch for now otherwise the visualizations won't load with DOJO AMD (ECL Watch).